frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -201,6 +201,7 @@ const TestCardForm = ({
         pregnancy: state.aoeResponses.pregnancy,
         genderOfSexualPartners:
           state.aoeResponses.genderOfSexualPartners ?? null,
+        testResultDelivery: testOrder.patient.testResultDelivery ?? null,
```

Comment:
**question** - why are we adding this back onto the mutation here? The new design for AOE questions doesn't allow the user to change delivery preferences outside of the `EditPatient` page. The `updateTimeOfTestQuestions` mutation is still being used by the old test card design so that's why the test result delivery logic in that mutation hasn't been removed yet, but I wonder if part of this bug is that the mutation is overriding the test result delivery preference as null when it isn't provided by the new test card, so perhaps we should add a null check for that here

```
@MutationMapping
  public void updateTimeOfTestQuestions(
      @Argument UUID patientId,
      @Argument String pregnancy,
      @Argument String symptoms,
      @Argument LocalDate symptomOnset,
      @Argument Boolean noSymptoms,
      @Argument List<String> genderOfSexualPartners,
      @Argument TestResultDeliveryPreference testResultDelivery) {

    Map<String, Boolean> symptomsMap = parseSymptoms(symptoms);

    testOrderService.updateTimeOfTestQuestions(
        patientId, pregnancy, symptomsMap, symptomOnset, noSymptoms, genderOfSexualPartners);

    if (testResultDelivery != null) {
      personService.updateTestResultDeliveryPreference(patientId, testResultDelivery);
    }
  }
```

Reply:
Yes! that's exactly what was happening! I like addressing this on the backend. Thank you for the question/suggestion mike! Will update this on `updateTimeOfTestQuestions` and `addPatientToQueue` as well. :)